hack/golint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
```

Comment:
Double quotes around curly brackets?

Reply:
ok

Reply:
Check [this](https://github.com/metal3-io/cluster-api-provider-metal3/blob/036ea1866c391e8d22213cfc7c0ebba597292b1d/hack/golint.sh#L5) as well